src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/Header.tsx

```diff
@@ -171,17 +232,23 @@ const Header: FC<HeaderProps> = ({
 				</TitleBar>
 				<Row nogutter align="center">
 					{isPendingApproval && isAdmin && (
-						<Button
-							id="button-reopen"
-							variant={isAdmin ? 'secondary' : 'text'}
-							isAsync
-							css={css`
-								margin-right: 10px;
-							`}
-							onClick={handleSubmissionReopen}
-						>
-							reopen submission
-						</Button>
+						<>
+							<Button
+								id="button-reopen"
+								variant={isAdmin ? 'secondary' : 'text'}
+								isAsync
+								css={css`
+									margin-right: 10px;
+								`}
+								onClick={handleSubmissionReopen}
+							>
+								reopen submission
+							</Button>
+
+							<Button id="button-approve" size="sm" isAsync onClick={handleSubmissionApproval}>
+								approve
```

Comment:
Just want to validate if these should be capitalized? 'Approve' + 'Reopen Submission' copy

Reply:
They get capitalised by the `Button` but it's a good point that this is not obvious. I will fix.

Reply:
https://github.com/icgc-argo/rdpc-ui/pull/120/commits/d9e5ebea31090688b85e125d8fa357b90aa10429